Formula/tre-command.rb

```diff
@@ -14,6 +14,7 @@ def install
   end
 
   test do
-    system "#{bin}/tre", testpath
+    (testpath/"foo.txt").write("")
+    assert(shell_output("#{bin}/tre").include?("── foo.txt"))
```

Comment:
Could you use this style, it's more in line with the other formula
```suggestion
    assert_match "foo.txt", shell_output("#{bin}/tre")
```